From c91b970f601178ac7edaf620a9daddfd6aad8afe Mon Sep 17 00:00:00 2001
From: Xiretza <xiretza@xiretza.xyz>
Date: Sun, 15 May 2022 11:15:01 +0200
Subject: [PATCH 1/2] fix(omp): read version information file in text mode

This was already fixed for the other bindings in
5e9aa4d3d4d3f476ab32fa10a08f28a56495e8f4, but omp was missed.
---
 omp/setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/omp/setup.py b/omp/setup.py
index 942ab1f5..5281398e 100755
--- a/omp/setup.py
+++ b/omp/setup.py
@@ -25,7 +25,7 @@ class GNATCollOMP(SetupApp):
 
         # Set library version
         with open(os.path.join(config.source_dir, '..',
-                               'version_information'), 'rb') as fd:
+                               'version_information'), 'r') as fd:
             version = fd.read().strip()
         config.set_data('GNATCOLL_VERSION', version, sub='gprbuild')
 
-- 
2.36.1

